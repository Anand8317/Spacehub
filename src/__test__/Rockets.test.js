import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import RocketsList from '../components/Rockets';
import store from '../redux/configureStore';

it('rockets list renders properly', () => {
  const tree = renderer
    .create(<Provider store={store}><RocketsList /></Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
