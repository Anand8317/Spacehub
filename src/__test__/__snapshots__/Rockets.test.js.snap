// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    className="rocketContainer"
  >
    <img />
    <div
      className="rocketInfo"
    >
      <h3 />
      <p>
        <span
          id="not-reserved"
        />
      </p>
      <button
        className="reserveBtn"
        id="reserve"
        onClick={[Function]}
        type="button"
      >
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;

exports[`rockets list renders properly 1`] = `
<div>
  <div
    className="rocketContainer"
  >
    <img />
    <div
      className="rocketInfo"
    >
      <h3 />
      <p>
        <span
          id="not-reserved"
        />
      </p>
      <button
        className="reserveBtn"
        id="reserve"
        onClick={[Function]}
        type="button"
      >
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;
